packages/megasync-deb/megasync-deb.pacscript

```diff
@@ -0,0 +1,38 @@
+name="megasync-deb"
+gives="megasync"
+repology=("project: megasync")
+version="4.9.4"
+build_version="-3.1"
+replace=("${gives}")
+breaks=("${gives}" "${gives}-bin" "${gives}-git" "${gives}-app" "${gives}-deb")
+incompatible=("debian:sid" "*:buster" "*:stretch" "*:jessie" "debian:testing" "*:bionic" "*:focal" "*:kinetic" "*:mantic")
+arch=("amd64")
+
+case "${DISTRO#*:}" in
+    lunar)
+        distro_base="xUbuntu"
+        distro_release="23.04"
+        hash="e1c6d66b9946d0e5a427560a39c0b2affe72714b57b3a41ceefac8304a0a3113"
+        ;;
+    jammy)
+        distro_base="xUbuntu"
+        distro_release="22.04"
+        hash="35a22daee1b9b5b8b2c619ebf80614a9781314cc1c5a3ad80d55af9d6cd6e378"
+        ;;
+    bookworm)
+        distro_base="Debian"
+        distro_release="testing"
+        hash="6fc4eda65d253bb42d4550e6a80fa0a6eda15978fe6dcebb61ff3e7a38c55096"
+        ;;
+    bullseye)
+        distro_base="Debian"
+        distro_release="11"
+        hash="929189eeaed61ce33efba1a3ead6159cde9b9601147a68854345a1b9b955cda5"
+        ;;
+    *) ;;
```

Comment:
It's customary to put something here just to be safe, usually just a `return 1`.[^1]

[^1]: https://github.com/pacstall/pacstall-programs/blob/9cb071b064bd6d5dc57c7164ce05d5401b886d66/packages/mold-bin/mold-bin.pacscript#L22

Reply:
If I add the `return 1` and the script runs on an incompatible distro instead of "incompatible distro" message, the pacstall throws "failed sourcing package" which I feel happens in 3.14.1. and was not the case in older version 3.13.x

Reply:
Oh true lol don't listen to me 👀